README.md

```diff
@@ -1,4 +1,7 @@
-## RPC Endpoint
+## Flashbots RPC Endpoint (_flashbots-rpc-endpoint_)
+
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
```

Comment:
Yes. I want more and more projects to follow this standard so it will become easier to jump into a new project, I've been pushing this for years :) For that to happen, it's good to give credit and to make it clear for others enjoying your README that they can also follow this spec.